sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +124,71 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs "binDir datDir" ./configure \
```

Comment:
* Suggestion (remove empty line before `defineDebugInfoPackage`)
* Typo fix for `-omit-dirs` (`datDir` :arrow_right: `dataDir`)

```diff
 	)
 fi
-
 defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure --omit-dirs "binDir datDir" ./configure \
+	runConfigure --omit-dirs "binDir dataDir" ./configure \
 		--bindir "$commandBinDir" \
 		--datadir="$dataDir"/gdal1 \
```

Reply:
Nice catch (already made the typo first with $dataDir/gdal1 :) comming up :)